src/value.rs

```diff
@@ -24,4 +24,8 @@ pub trait Value: Clone + Default + Debug + From<FunctionRef<Self::TS>> + Partial
 
     /// Assign to this value
     fn assign(&mut self, value: <Self::TS as TypeSystem>::Value);
+
+    #[cfg(feature="variadic_functions")]
+    /// Create a `Value` type list out of `Vec` of `Value`
+    fn gen_list(values: Vec<Self>)->Self;
```

Comment:
Formatting is off

Reply:
what are you talking about?

Reply:
`)->Self;` should be `) -> Self;`

Reply:
oh nvm I see it, but that was already handled on a different branch